styleguide.config.js

```diff
@@ -1,4 +1,17 @@
 const path = require('path')
+const CopyWebpackPlugin = require('copy-webpack-plugin')
```

Comment:
Is there any reason this file lives in the root and not in `webpack` alongside the other config files?

Reply:
Just to be clear the file `styleguide.config.js` ... expected it to be something like `webpack/styleguide.config.js`

Reply:
Does your comment relate to this line of the file?

Or about the location of this whole file?

Reply:
The whole file

Reply:
I think this whole file needs to stay in the root. But we are right now moving just the webpack portion of this whole styleguidist config file into the `/webpack` directory.

Does that sound better?